plugins/alias-finder/README.md

```diff
@@ -0,0 +1,48 @@
+# alias-finder plugin
+
+## Description
+This plugin searches the defined aliases and outputs any that match the command inputted. This makes learning new aliases easier.
+
+## Enabling
```

Comment:
The Description and Enabling sections are normally merged together, without any header:
```markdown
# plugin title

Words of the description

Instructions to enable
```